.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### User Story
+As a <role: Devloper, SysAdmin, etc.>, I will use this work when I want to ...
+
+## Description
+<!--- Briefly describe the issue -->
+
+## Chef Version
+<!--- Tell us which version of chef-client, chef-server+ChefDK you need docs for) -->

```

Comment:
`chef-client, chef-server+ChefDK` should be `chef-client, chef-server, or chef dk`?

Reply:
I generally stylize it as ChefDK, as distinct from `chef-dk` which is the name of a gem.